src/data/finset/basic.lean

```diff
@@ -2716,47 +2760,11 @@ begin
     simp only [disjoint_union_left, bUnion_insert, his, forall_mem_insert, ih] }
 end
 
-lemma disjoint_bUnion_right {ι : Type*} (s : finset α) (t : finset ι) (f : ι → finset α) :
+lemma disjoint_bUnion_right (s : finset β) (t : finset α) (f : α → finset β) :
```

Comment:
This is dangerous because we might assume `decidable_eq α` but we never assume `decidable_eq ι`. It is customary to use `ι` for indexed families, so I would be inclined to keep it as is/move `ι` to the first `variables`.

Reply:
This change is to match the rest of the `bUnion` section. The diff doesn't show it well, but these lemmas are now part of that section.